app/assets/javascripts/admin/app/controllers/gobierto_attachments_controller.js

```diff
@@ -431,10 +437,35 @@ this.GobiertoAdmin.GobiertoAttachmentsController = (function() {
             success: function(response, textStatus, jqXHR){
               if(jqXHR.status == 200){
                 Vue.set(self, 'attachments', response.attachments);
+
+                if(response.attachments.length > 0) {
+                  self.showFiles = true;
+                  self.fileListClass = 'fa-caret-down';
+                } else {
+                  self.showFiles = false;
+                  self.fileListClass = '';
+                }
               }
-            },
+            }
           });
         },
+        modifyfileListClass: function(){
+          var self = this;
```

Comment:
There's no need to assign this to self when there's no issue with the binding. You can use `this` in this method.